README.md

```diff
@@ -1,10 +1,27 @@
 # Firefox Localizations
 
-This is a monorepo replacement for the [l10n-central](https://hg.mozilla.org/l10n-central/) repositories.
+This repository contains the localized messages for all Firefox locales.
 
-While we're transitioning from hg to git,
+If you’d like to submit translations or suggest improvements to existing
+strings please visit us on [Pontoon](https://pontoon.mozilla.org/projects/firefox/).
+Changes to translations will not be accepted by pull request
+(with exceptions for specific locales).
+
+## String updates
+
+This repo contains only localized files;
+source strings are located in the 
+[firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source/?tab=readme-ov-file)
```

Comment:
```suggestion
[firefox-l10n-source](https://github.com/mozilla-l10n/firefox-l10n-source/)
```

What's the purpose of the extra parameter?

Reply:
The base repo view can have additional tabs, such as: https://github.com/mozilla-l10n/firefox-l10n?tab=MPL-2.0-1-ov-file

The readme is the default, and unnecessary here.